bin/infrasim-console

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+'''
```

Comment:
@fub2 I know what arys does.  But I prefer to move them into infrasim repo, so they seems to be  a part of infrasim application